chromium_src/chrome/common/extensions/chrome_extensions_client.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_COMMON_EXTENSIONS_CHROME_EXTENSIONS_CLIENT_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_COMMON_EXTENSIONS_CHROME_EXTENSIONS_CLIENT_H_
+
+#define ChromeExtensionsClient ChromeExtensionsClient_ChromiumImpl
+#include "../../../../../chrome/common/extensions/chrome_extensions_client.h"
+#undef ChromeExtensionsClient
+
+namespace extensions {
+
+class ChromeExtensionsClient : public ChromeExtensionsClient_ChromiumImpl {
```

Comment:
why not we add a fully-functioning `BraveExtensionsClient` and override only instantiation spot? @bridiver

Reply:
added: https://github.com/brave/brave-core/pull/9933/files#diff-40662399037b3c7f1c12c31b0161cf86dcb72542716c8e425a9a79b8a2cad470